todo_app/templates/index.html

```diff
@@ -10,7 +10,12 @@ <h1 class="display-4">To-Do App</h1>
     <form action="http://127.0.0.1:5000/addNewTitle" method="post">
         <label for="title">Enter your title: </label>
         <p><input type="text" name="nm"></p>
-        <p><input type = "submit" value = "submit" /></p>    
+        <p><input type = "submit" value = "submit" /></p>
+        <label for="title">Update your card: </label>
+        <p><input type="text" id="cardId"></p>
+        <input type="button" onclick="window.location.href='/updateCard/60afa040b07e9a4371098530/ToDo';" value="ToDo"/>
+        <input type="button" onclick="window.location.href='/updateCard/60afa040b07e9a4371098530/Doing';" value="Doing"/> 
+        <input type="button" onclick="window.location.href='/updateCard/60afa040b07e9a4371098530/Done';" value="Done"/> 
```

Comment:
Here you hardcode a card id, so you're always updating the same fixed card
Instead of using the path, we could read the `cardId` input in your `updateCard` function as you've already set that up

A nicer user experience is likely to be a separate button per card though; that is easier than copy-pasting ids around, and we could use the path like you try here inside your for loop

Reply:
Actioned